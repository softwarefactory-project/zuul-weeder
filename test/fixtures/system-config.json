{
    "system_attributes": {
        "default_ansible_version": null,
        "default_hold_expiration": 86400,
        "max_hold_expiration": 0,
        "use_relative_priority": false,
        "web_root": "https://sftests.com/zuul/t/{tenant.name}/",
        "web_status_url": "",
        "websocket_url": null
    },
    "unparsed_abide": {
        "admin_rules": [
            {
                "conditions": [
                    {
                        "groups": "config-ptl"
                    },
                    {
                        "groups": "config-core"
                    }
                ],
                "name": "config-acl"
            }
        ],
        "tenants": {
            "local": {
                "max-job-timeout": 10800,
                "name": "local",
                "report-build-page": true,
                "default-parent": "base",
                "source": {
                    "gerrit": {
                        "config-projects": [
                            {
                                "config": {}
                            }
                        ],
                        "untrusted-projects": [
                            {
                                "sf-jobs": {
                                    "exclude": "pipeline"
                                }
                            },
                            {
                                "zuul-jobs": {
                                    "include": [
                                        "job"
                                    ],
                                    "shadow": "sf-jobs"
                                }
                            },
                            {
                                "zuul-distro-jobs": {
                                    "include": []
                                }
                            }
                        ]
                    }
                }
            }
        },
        "uuid": "b748d19f1c674b29b728d81e79520635"
    }
}