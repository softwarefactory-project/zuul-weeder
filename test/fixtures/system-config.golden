[
    ( TenantName
        { getName = "local" }
    , TenantConfig
        { defaultParent = JobName "base"
        , connections = fromList
            [
                ( ConnectionName "gerrit"
                , TenantConnectionConfig
                    { configProjects =
                        [ TenantProject
                            { projectName = ProjectName "config"
                            , includedConfigElements = fromList
                                [ PipelineT
                                , JobT
                                , SemaphoreT
                                , ProjectT
                                , ProjectTemplateT
                                , NodesetT
                                , SecretT
                                , QueueT
                                ]
                            , configPaths =
                                [ ".zuul.yaml"
                                , "zuul.yaml"
                                , ".zuul.d/"
                                , "zuul.d/"
                                ]
                            }
                        ]
                    , untrustedProjects =
                        [ TenantProject
                            { projectName = ProjectName "sf-jobs"
                            , includedConfigElements = fromList
                                [ JobT
                                , SemaphoreT
                                , ProjectT
                                , ProjectTemplateT
                                , NodesetT
                                , SecretT
                                , QueueT
                                ]
                            , configPaths =
                                [ ".zuul.yaml"
                                , "zuul.yaml"
                                , ".zuul.d/"
                                , "zuul.d/"
                                ]
                            }
                        , TenantProject
                            { projectName = ProjectName "zuul-jobs"
                            , includedConfigElements = fromList [ JobT ]
                            , configPaths =
                                [ ".zuul.yaml"
                                , "zuul.yaml"
                                , ".zuul.d/"
                                , "zuul.d/"
                                ]
                            }
                        , TenantProject
                            { projectName = ProjectName "zuul-distro-jobs"
                            , includedConfigElements = fromList []
                            , configPaths =
                                [ ".zuul.yaml"
                                , "zuul.yaml"
                                , ".zuul.d/"
                                , "zuul.d/"
                                ]
                            }
                        ]
                    }
                )
            ]
        }
    )
]